### PR TITLE
add com.gigitux.gtkwhats

### PR DESCRIPTION
**Pull request by gigitux**

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build com.gigitux.gtkwhats

**Comment by gigitux** [comment]
@LeandroStanger  Do i remove this option?

```
"build-args": ["--share=network"],
```

**Comment by gigitux** [comment]
@LeandroStanger Updated :+1:

**Comment by gigitux** [comment]
bot, build com.gigitux.gtkwhats

**Comment by gigitux** [comment]
bot, build com.gigitux.gtkwhats

File: com.gigitux.gtkwhats.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "gtkwhats",
+  "finish-args": [
+    "--share=ipc",
+    "--share=network",
+    "--socket=fallback-x11",
+    "--socket=wayland",
+    "--socket=session-bus",
```
Comment (bilelmoussaoui) [inline com.gigitux.gtkwhats.json:13]: What do you need the session-bus for?

File: com.gigitux.gtkwhats.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    "--device=dri",
+    "--socket=pulseaudio"
+  ],
+  "build-options": {
+    "append-path": "/usr/lib/sdk/rust-stable/bin",
+    "env": {
+      "RUSTFLAGS": "--remap-path-prefix =../",
```
Comment (bilelmoussaoui) [inline com.gigitux.gtkwhats.json:20]:
```suggestion
```
you don't need this anymore

File: com.gigitux.gtkwhats.json (hunk 1 above)
Comment (gigitux) [reply com.gigitux.gtkwhats.json:13]: I need this to make notifications from WebKit work

Comment (bilelmoussaoui) [reply com.gigitux.gtkwhats.json:13]: you need `--talk-name=org.freedesktop.Notifications` instead

Comment (gigitux) [reply com.gigitux.gtkwhats.json:13]: I have this error when open application:
```
Failed to register: GDBus.Error:org.freedesktop.DBus.Error.ServiceUnknown: org.freedesktop.DBus.Error.ServiceUnknown
```

Comment (bilelmoussaoui) [reply com.gigitux.gtkwhats.json:13]: Your issue comes from https://github.com/gigitux/gtkwhats/blob/master/src/constants/constants.rs#L3. The app_id you pass to the gtk::Application should be the same as the one you define on your flatpak manifest

**Comment by bilelmoussaoui** [comment]
bot, build com.gigitux.gtkwhats

**Comment by barthalion** [comment]
bot, build com.gigitux.gtkwhats

**Comment by barthalion** [comment]
Icon seems to be missing, both in `About` and when switching windows in GNOME.

**Comment by gigitux** [comment]
@barthalion 

I was aware of the lack of the icon on the about page.
I would like to share the project as soon as it is uploaded to flathub and see if some designer makes me a custom logo.

As for the "switching windows" I see the icon. Can you make me a screen?

Thanks a lot!

**Comment by barthalion** [comment]
Hm, looks like some cache wasn't refreshed or something, it's there now.

/merge

**Pull request closed without merging**